pages/asks.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useMemo, ReactElement } from "react";
+import styles from "../styles/pages/asks.module.scss";
+import content from "../server/asks";
+import Ask from "../components/Ask";
+import CreateAskForm from "../components/CreateAskForm";
+import Modal from "../components/Modal";
+
+interface AskContent {
```

Comment:
convert it to a type alias

Reply:
wait... why do you define it here if it is already defined in the content file...?
and it's the same type aswell :')
kill this

Reply:
noted